test/run-tests.py

```diff
@@ -752,18 +750,16 @@ def main(args):
   except KeyboardInterrupt:
     print('\nInterrupted testing\n')
 
-  status.Clear()
+  status.Done()
 
   ret = 0
-
   if status.failed:
     sys.stderr.write('**** FAILED %s\n' % ('*' * (80 - 14)))
     for info in status.failed_tests:
       sys.stderr.write('- %s\n    %s\n' % (info.GetName(),
                                            ' '.join(info.last_cmd)))
     ret = 1
 
-  status.Print()
```

Comment:
Hm, well I'm running this on a machine with `multiprocessing.cpu_count() => 32`, maybe that makes a difference?